openmetadata-docs/content/connectors/pipeline/domo-pipeline/index.md

```diff
@@ -40,6 +40,13 @@ To deploy OpenMetadata, check the <a href="/deployment">Deployment</a> guides.
 To run the Ingestion via the UI you'll need to use the OpenMetadata Ingestion Container, which comes shipped with
 custom Airflow plugins to handle the workflow deployment.
 
+<Note>
+
+For metadata ingestion, kindly make sure add alteast `data` scopes to the clientId provided.
```

Comment:
Do pipelines and databases share the same scope?

Reply:
yes